sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-process-integration.scheduler.models.ts

```diff
@@ -23,6 +29,26 @@ export interface MSFAATestInputData {
   offeringIntensity: OfferingIntensity;
 }
 
+export const MSFAA_FULL_TIME_MARRIED: MSFAATestInputData = {
+  msfaaNumber: "2000",
```

Comment:
Are the goal od the cont like `PART_TIME_SAMPLE_MSFAA_NUMBER` to be used here also?

Reply:
I did not change the `msfaaNumber` in the above line to use the constant `PART_TIME_SAMPLE_MSFAA_NUMBER` because I had not created the constant for use here. However, I can change the code to use the above constant.

Reply:
As we talked, the constant was removed and that will also eliminate possible confusion in the future. It is ok for me.